CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- The `SaferAlterFieldSetNotNull` operation was added. This will safely change
+  a field from nullable to not nullable.
```

Comment:
🐼 safely -> more safely - (nothing is 100% safe)

Reply:
922d2c84de74a3539d53460047253d026dd82c79